cosmic-settings/src/pages/power/mod.rs

```diff
@@ -85,7 +85,7 @@ fn profiles() -> Section<crate::pages::Message> {
                     section = section.add(item);
                 }
             } else {
-                let item = widget::Text::new(fl!("power-profiles", "nobackend"));
+                let item = widget::Text::new(fl!("power-mode", "nobackend"));
```

Comment:
This should be `widget::text::body`

Reply:
I'm not very familiar with this, but when compiling, it says that the `body` function isn't found in module `widget::text`. Any tips as to what should be imported instead of `cosmic::widget::text`?

Reply:
You can see available text variants here: https://pop-os.github.io/libcosmic/cosmic/widget/text/index.html
The body function does exist here.

Reply:
This module is importing `use cosmic::iced::widget;` when it should be importing `cosmic::widget`

Reply:
It seems to compile if a change that line to `cosmic::widget::text::body(...)`, but I'm likely not importing something correctly.

Reply:
Another issue is that the update method is blocking the runtime, when it should be spawning commands.

Reply:
I'll push the variant that compiles so that the rest of the changes can be reviewed.

Reply:
Oh wow, your responses didn't show up at all until the site was refreshed. Sorry for seemingly ignoring your answers.

Reply:
>Another issue is that the update method is blocking the runtime, when it should be spawning commands.

What should I do about this?

Reply:
The update method needs to be restructured to return `cosmic::command`s instead of blocking.

Reply:
I'm not sure that I'm capable of doing that. :(

Reply:
Then I'll do it once this is merged